packages/@tailwindcss-node/src/compile.ts

```diff
@@ -45,3 +46,19 @@ export async function compile(
     },
   })
 }
+
+// Attempts to import the module using the native `import()` function. If this
+// fails, it sets up `jiti` and attempts to import this way so that `.ts` files
+// can be resolved properly.
+let jiti: null | Jiti = null
+async function importModule(path: string): Promise<any> {
+  try {
+    return await import(path)
+  } catch (error) {
+    try {
+      jiti ??= createJiti(import.meta.url, { moduleCache: false, fsCache: false })
+      return await jiti.import(path)
+    } catch {}
+    throw error
+  }
+}
```

Comment:
I _think_ we normally try to declare things in the order they are used, so we'd normally put this stuff at the top of the file. Not dogmatic about either approach but would like us to be consistent with one or the other 👍

Reply:
Can you clarify what you mean with "in order they are used"? My thinking is to write functions in the order that you would read them, you start by the exports and "high level" modules and then you see "oh this function uses `importModule` which is a lower level implementation detail so it doesn't need to be at the very top. Happy to changer to whatver, I’m just not sure I follow what you mean by "order they are used". Used by whom?

Reply:
We noticed that we are already not consistent here so this isn't blocking.